utils/cr_demo/run_demo.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+dmg_pool_list=`dmg pool list`
+dmg_check_enable=`dmg check enable`
+dmg_check_start=`dmg check start`
+dmg_check_query=`dmg check query`
+dmg_check_disable=`dmg check disable`
+POOL_SIZE="4G"
+POOL_LABEL="tank"
+
+format_storage() {
+    hostlist=$1
+    dmg_cmd=`dmg storage format --host-list=$hostlist`
+    dmg_cmd
+}
+
+create_pool() {
+    pool_size=$1
+    pool_label=$2
+    dmg_cmd=`dmg pool create --size=$pool_size --label=$pool_label`
+    dmg_cmd
+}
+
+inject_fault_mgmt() {
+    pool_label=$1
+    fault_type=$2
+    dmg_cmd=`dmg fault mgmt-svc pool $pool_label $fault_type`
```

Comment:
(lint) dmg_cmd appears unused. Verify use (or export if used externally). [SC2034]
(lint) Use $(...) notation instead of legacy backticked `...`. [SC2006]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]